### PR TITLE
Password hashing

### PR DESCRIPTION
**Pull request by FalseR20**

### CODE REVIEW

File: sc_server_auth/create_admin.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from getpass import getpass
+
+from sc_server_auth.configs.log import get_default_logger
+from sc_server_auth.configs.models import Database
+from sc_server_auth.configs.parser import get_config
+from sc_server_auth.server.admin import verify_user_info_in_database
+from sc_server_auth.server.database import DataBase
```
Comment (kroschenko) [inline sc_server_auth/create_admin.py:9]: I think, it's not good to use same names (database), especially if they differ by letter

File: sc_server_auth/server/admin.py [hunk 2]
```diff
@@ -14,7 +15,7 @@
 )
 
 
-def _verify_user_info_in_database(database: DataBase, name: str, password: str) -> m.ResponseModel:
+def verify_user_info_in_database(database: DataBase, name: str, password: str) -> m.ResponseModel:
```
Comment (Dzinushi) [inline sc_server_auth/server/admin.py:18]: why this function is public now?

File: sc_server_auth/server/hashing.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import hashlib
+
+_HASH_NAME = "sha256"
+_SALT = b"\xf2\x0fr{\x98\x12\x9f\x08'\xfd_\x1f-sF\x84"
```
Comment (kroschenko) [inline sc_server_auth/server/hashing.py]: Is it really a good idea store salt here?

**Review by Dzinushi — APPROVED** [review]

File: sc_server_auth/create_admin.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
+from argparse import ArgumentParser
```
Comment (kroschenko) [inline sc_server_auth/create_admin.py:1]: Who can add admin now? Do I understand correctly what this script can be executed only locally on the server?

**Review by kroschenko — CHANGES_REQUESTED** [review]

File: sc_server_auth/server/admin.py (hunk 2 above)
Comment (FalseR20) [reply sc_server_auth/server/admin.py:18]: It's used in create_admin.py

File: sc_server_auth/create_admin.py (hunk 4 above)
Comment (FalseR20) [reply sc_server_auth/create_admin.py:1]: Yes, like manage.py in django

File: sc_server_auth/server/hashing.py (hunk 3 above)
Comment (FalseR20) [reply sc_server_auth/server/hashing.py]: It can be in toml settings file

File: sc_server_auth/create_admin.py (hunk 1 above)
Comment (FalseR20) [reply sc_server_auth/create_admin.py:9]: You're right, I renamed Database on DatabaseType

File: sc_server_auth/server/hashing.py (hunk 3 above)
Comment (FalseR20) [reply sc_server_auth/server/hashing.py]: Done

**Review by kroschenko — APPROVED** [review]

**Pull request merged**